src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedMathUtils.java

```diff
@@ -0,0 +1,437 @@
+package org.broadinstitute.hellbender.utils;
```

Comment:
I think you restored this class by mistake.  It has been absorbed into `MathUtils`, perhaps between rebases.

Reply:
Seems like I was referencing `sumArrayFunction` from this deprecated class, and it didn't get deleted during rebase. I removed the class and updated the function call to `MathUtils.sumArrayFunction`